src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessage {
+  none = '',
```

Comment:
```suggestion
  None = '',
```
first letter in key should be capitalized